src/Uuid/UuidGenerator.php

```diff
@@ -33,12 +33,12 @@ public function generateV4() {
      *
      * @return String   Valid v5 UUID.
      */
-    public function generateV5($str, $namespace = NULL) {
+    public function generateV5($namespace = NULL) {
         // Use default namespace if none is provided.
         if (empty($namespace)) {
             $namespace = $this->namespace;
         }
 
-        return Uuid::uuid5($namespace, $str)->toString();
+        return Uuid::uuid5(Uuid::NAMESPACE_DNS, $namespace)->toString();
```

Comment:
Dear people, i think you have that uuid v5 implementation wrong.

UUIDv5 depends on a previous generated UUID. So there are two ones, the first uses `Uuid::NAMESPACE_DNS`, the second one is based on the first. Please look at:

https://github.com/Islandora-CLAW/CLAW/blob/sprint-002/services/ResourceService/src/Provider/UUIDServiceProvider.php#L20-L37